src/modal/modal.js

```diff
@@ -28,6 +28,9 @@ angular.module('mgcrea.ngStrap.modal', ['mgcrea.ngStrap.core', 'mgcrea.ngStrap.h
       var requestAnimationFrame = $window.requestAnimationFrame || $window.setTimeout;
       var bodyElement = angular.element($window.document.body);
 
+      var modalCount = 0;
+      var baseZindexValue = 1037;
```

Comment:
Any reason you picked this specific number? Looks like it should default to 1050 according to https://github.com/twbs/bootstrap/blob/master/less/variables.less#L277